server/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const db = require('./db');
+const app = express();
+const port = 3000;
+
+
+app.get('/items/:itemId/reviews', (req, res) => {
+  db.getAllReviewsForItem(1, (err, data) => {
+    if (err) {
+      console.log('error getting item reviews: ', err);
+    } else {
+      var parsedData = JSON.parse(data)
+      console.log('success getting reviews for item');
```

Comment:
console logs are good for development, but I've made a practice of removing them all after I've tested functionality is working, before pushing to github.  Up to you!

Reply:
since this is a console log in the terminal I'm not too worried about it, but thanks for the suggestion! I am mostly leaving this in since I will likely be messing with this more in the future.